manage/roles/admin-permissions.rst

```diff
@@ -0,0 +1,167 @@
+          See `Issue 3194 <https://github.com/zammad/zammad/issues/3194>`_ for more.
+
+``admin.channel_facebook``
+   Grants access to the :doc:`/channels/facebook` settings page.
+
+      .. note:: Access to facebook based tickets depends on agents :doc:`group permissions <group-permissions>`.
+
+``admin.channel_formular``
+   Grants access to the :doc:`/channels/form` setting page.
+
+``admin.channel_sms``
+   Grants access to the SMS settings page.
+
+``admin.channel_telegram``
+   Grants access to the :doc:`/channels/telegram` settings page.
+
+      .. note:: Access to telegram based tickets depends on agents :doc:`group permissions <group-permissions>`.
+
+``admin.channel_twitter``
+   Grants access to the :doc:`/channels/twitter` settings page.
+
+      .. note:: Access to twitter based tickets depends on agents :doc:`group permissions <group-permissions>`.
+
+``admin.channel_web``
+   Grants access to the :doc:`/channels/web` settings page.
+
+``admin.data_privacy``
+   Provides permission to :doc:`/system/data-privacy`. 
+
+      .. warning:: ⚠ This allows removal of data and thus is a potential dangerous permission!
+
+``admin.group``
+   Allows users to add or change :doc:`/manage/groups` and their settings.
+
+``admin.integration``
+   This permission grants access **to all** available :doc:`/system/integrations`.
+
+``admin.knowledge_base``
+   Grants access to the base configuration of the :doc:`/manage/knowledge-base`.
+
+      .. hint:: 🤓 The knowledge base access (read and edit) depends on :doc:`agent-permissions` and 
+         the `answer visibility <https://user-docs.zammad.org/en/latest/extras/knowledge-base.html#editing-answers>`_.
+
+``admin.macro``
+   Allows management of :doc:`/manage/macros`. 
+
+      .. note:: Macros may indirectly require access to :doc:`/manage/tags` in some situations.
+
+``admin.maintenance``
+   Grants access to :doc:`maintenance settings </system/maintenance>`.
+
+``admin.monitoring``
+   This permission access to Zammads :doc:`/system/monitoring` page.
+
+``admin.object``
+   Grants access to :doc:`object management </system/objects>`.
+
+``admin.organization``
+   Grants access the :doc:`organization management </manage/organizations>`.
+
+      .. note:: Agents can always find and edit existing organizations. 
+         **Exception:** Agents can't set ``shared organization`` or ``domain based assignment``.
+
+``admin.overview``
+   Allows users to manage :doc:`/manage/overviews` for users.
+
+``admin.package``
+   Provides access to the :doc:`package management </system/packages>` for Zammad addons.
+
+``admin.report_profile``
+   Allows users to define :doc:`/manage/report-profiles` to enhance their Reports.
+
+      .. hint:: 🤓 Report profiles are relevant for the Reporting.
+         Access to the Reporting function depends on :doc:`agent-permissions`.
+
+      ..tip:: This is a great permission for key users. 🚀
```

Comment:
Sorry, I did not get what you meant by "key" users, so I deleted the two or three lines that contained notes like this.

Reply:
No worries.
I'm not sure if there's an international wording for this:

A key user in germany is a user that has a better technical understand than the other users.
This user is not an administrator but has a greater understanding of the software in use.

German companies use key users in order to spread knowledge to other users.
This helps to reduce the number of people that you have to send to a workshop and also allows you to keep on working if you send users to a workshop.

Because if the whole company goes to a workshop, that'd mean you'd have to take a "day off" support which in some cases doesn't help. I'm fine with leaving this wording away if it doesn't fit!

Reply:
It sounds like what you're describing is a “team lead”? Conversationally/half-humorously, you may refer to someone like this as the “resident expert” on a subject (but I definitely wouldn't use that term here). But to be honest, I haven't spent enough time in large teams in the workplace to know if there is really a comparable term in English.

More broadly, it sounds like we're describing another role between Agent and Admin—sort of a Super Agent? My take is that it would be better to illustrate this concept by actually having this sample role included by default in every instance of Zammad, rather than adding a note to every relevant permission in the reference docs.

Reply:
Okay. Let's just skip this part then for now. It's not super important. :)